domain-models-runtime/src/main/java/org/folio/rest/persist/PostgresClient.java

```diff
@@ -1709,10 +1706,32 @@ public TotaledResults(RowSet<Row> set, Integer total) {
     Handler<AsyncResult<PostgresClientStreamResult<T>>> replyHandler) {
 
     streamGet(table, clazz, fieldName, filter, returnIdField, distinctOn,
-      null, replyHandler);
+      null, 0, replyHandler);
   }
 
-  /**
+   /**
+   * Stream GET with CQLWrapper, no facets {@link org.folio.rest.persist.PostgresClientStreamResult}
+   * @param <T>
+   * @param table
+   * @param clazz
+   * @param fieldName
+   * @param filter
+   * @param returnIdField
+   * @param distinctOn may be null
+   * @param replyHandler AsyncResult; on success with result {@link org.folio.rest.persist.PostgresClientStreamResult}
+   * @param queryTimeout query timeout in milliseconds, or 0 for no timeout
+   */
+   @SuppressWarnings({"squid:S00107"})    // Method has >7 parameters
+  public <T> void streamGet(String table, Class<T> clazz, String fieldName,
+      CQLWrapper filter, boolean returnIdField, String distinctOn,
+      Handler<AsyncResult<PostgresClientStreamResult<T>>> replyHandler,
+      int queryTimeout) {
```

Comment:
Can you switch the replyHandler and queryTimeout parameters?
It's much easier to read the method invocation when the muli-line replyHandler is at the end.

Reply:
done

Reply:
up